src/object.rs

```diff
@@ -337,6 +337,7 @@ pub trait Cast: ObjectType {
     /// Casts to `&T` unconditionally.
     ///
     /// Panics if compiled with `debug_assertions` and the instance doesn't implement `T`.
+    #[cfg_attr(feature = "cargo-clippy", allow(clippy::transmute_ptr_to_ptr))]
```

Comment:
Here we should be able to cast too then, or not?

Reply:
Maybe yes, now not sure why I decide ignoring.